examples/02-HFSS/Waveguide_Filter.py

```diff
@@ -0,0 +1,186 @@
+"""
+HFSS: Inductive Iris waveguide filter
+----------------------------------------
+This example shows how to build and analyze a 4-pole
+X-Band waveguide filter using inductive irises.
+
+Note that this example uses the Python package SymPy to
+evaluate expressions.
+"""
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Perform required imports. Note the dependence on
+# SymPy is used to evaluate expressions. SymPy is not
+# a default requirement in PyAedt and is only compatible
+# with CPython > 3.6.
+
+import os
+import tempfile
+import pyaedt
+from pyaedt import general_methods
+
+###############################################################################
+# Launch AEDT
+# ~~~~~~~~~~~
+#
+
+
+###############################################################################
+# Define values for the waveguide iris filter.
+# ~~~~~~~~~~~~~~~~~~~~~~
+
+wgparams = {'l': [0.7428, 0.82188],
+            'w': [0.50013, 0.3642, 0.3458],
+            'a': 0.4,
+            'b': 0.9,
+            't': 0.15,
+            'units': 'in'}
+
+non_graphical = False
+new_thread = True
+
+
+project_folder = tempfile.gettempdir()
+project_name = project_folder + "\\" + general_methods.generate_unique_name("wgf", n=2)
+
+# Connect to an existing instance if it exits.
+hfss = pyaedt.Hfss(projectname=project_name + '.aedt',
+                   specified_version="2022.2",
+                   designname="filter",
+                   non_graphical=non_graphical,
+                   close_on_exit=True)
+
+# hfss.settings.enable_debug_methods_argument_logger = False  # Only for debugging.
+
+var_mapping = dict()  # Used by parse_expr to parse expressions.
+
+###############################################################################
+# Initialize design parameters in HFSS.
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+hfss.modeler.model_units = "in"  # Set to inches
+for key in wgparams:
+    if type(wgparams[key]) in [int, float]:
+        hfss[key] = str(wgparams[key]) + wgparams['units']
+        var_mapping[key] = wgparams[key]  # Used for expression parsing
+    elif type(wgparams[key]) == list:
+        count = 1
+        for v in wgparams[key]:
+            this_key = key + str(count)
+            hfss[this_key] = str(v) + wgparams['units']
+            var_mapping[this_key] = v  # Used to parse expressions and generate numerical values.
```

Comment:
Notice how this inline comment and the next have a period but not the two before it. Within any example, either periods should always be used or never be used in code comments. You can decide which you prefer. The goal is consistency within the example.